lib/galaxy_test/api/test_jobs.py

```diff
@@ -937,9 +937,97 @@ def test_job_build_for_rerun_list_list(self, history_id):
         )
         assert rerun_content == run_content
 
+    @pytest.mark.require_new_history
+    def test_get_inputs_and_outputs(self, history_id):
+        dataset_id = self.__history_with_ok_dataset(history_id)
+        inputs = json.dumps({"input1": {"src": "hda", "id": dataset_id}})
+        search_response = self._create_and_search_job(history_id, inputs, tool_id="cat1")
+        job_id = search_response.json()[0]["id"]
+        job_first_output_name, job_first_output_values = list(search_response.json()[0]["outputs"].items())[0]
+        # get the inputs of the job
+        job_response = self._get(f"jobs/{job_id}/inputs")
+        self._assert_status_code_is(job_response, 200)
+        job_first_input = job_response.json()[0]
+        # validate input response
+        assert job_first_input.get("name") == "input1"
+        assert job_first_input.get("dataset") == {"src": "hda", "id": dataset_id}
+        # get the outputs of the job
+        job_response = self._get(f"jobs/{job_id}/outputs")
+        self._assert_status_code_is(job_response, 200)
+        job_first_output = job_response.json()[0]
+        # validate output response
+        assert job_first_output.get("name") == job_first_output_name
+        assert job_first_output.get("dataset").get("id") == job_first_output_values.get("id")
+        assert job_first_output.get("dataset").get("src") == job_first_output_values.get("src")
```

Comment:
The endpoints "inputs" and "outputs" also had no tests. So I wrote this. Any suggestions on making the tests more comprehensive or is it enough?

Reply:
this is fine